### PR TITLE
LPS-130986 Create tests to render MetricsCalculatedInfo in InstanceListPage

### PR DESCRIPTION
**Pull request by ramonmsp**
Story: [LPS-94760](https://issues.liferay.com/browse/LPS-94760)

### CODE REVIEW

**Comment by ramonmsp** [comment]
ci:test:sf

**Comment by ramonmsp** [comment]
ci:test:relevant

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/test/components/instance-list-page/InstanceListPage.es.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ describe('The instance list card should', () => {
 			.fn()
 			.mockResolvedValue({data: {items, totalCount: items.length + 1}}),
 	};
-	let container, getByText;
+	let container, getByText, findByText;
```
Comment (rafaprax) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/test/components/instance-list-page/InstanceListPage.es.js:59]: @ramonmsp sort this alphabetically, please. :)

Comment (rafaprax) [reply modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/test/components/instance-list-page/InstanceListPage.es.js:59]: and the calls below :)

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by naraandrade** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/instance-list-page/InstanceListPageHeader.es.js [hunk 2]
```diff
@@ -47,6 +53,16 @@ export default function Header({
 	const {openModal} = useContext(ModalContext);
 	const previousCount = usePrevious(totalCount);
 
+	const previousFetchData = usePrevious(fetchData);
```
Comment (naraandrade) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/instance-list-page/InstanceListPageHeader.es.js:56]: @ramonmsp could you add a line between the definition and the usage of previousFetchData? Thanks :)

**Review by naraandrade — CHANGES_REQUESTED** [review]

**Comment by ramonmsp** [comment]
ci:test:sf

**Comment by ramonmsp** [comment]
ci:test:sf

**Comment by ramonmsp** [comment]
ci:test:relevant

**Review by naraandrade — APPROVED** [review]

**Review by jotascript — APPROVED** [review]

**Comment by ramonmsp** [comment]
ci:test:relevant

**Review by rafaprax — APPROVED** [review]

**Comment by rafaprax** [comment]
ci:forward

**Pull request closed without merging**